### ISSUE TITLE
Warning: UIKeyboardBoundsUserInfoKey is deprecated

### ISSUE THREAD

**Issue by leonardoamx** [ideashower/ShareKit#300]
In SHKTwitterForm.m, method -(void)keyboardWillShow:(NSNotification *)notification, the line
[[notification.userInfo valueForKey:UIKeyboardBoundsUserInfoKey ] getValue:&keyboardFrame];
I get a warning 'UIKeyboardBoundsUserInfoKey is deprecated'
using iOS SDK 4.3

**Comment by statico** [comment]
Useful until ShareKit is updated: http://stackoverflow.com/questions/194666/is-there-a-way-to-suppress-warnings-in-xcode

```
#pragma GCC diagnostic ignored "-Wwarning-flag"
```

**Comment by leonardoamx** [comment]
Does this disable warnings in the entire project or only in the class where it is used?

**Comment by leonardoamx** [comment]
I don't think that disabling warnings is a solution. If UIKeyboardBoundsUserInfoKey is deprecated, it must be another way to make this instruction.
